bin/ckan-merge-pr.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+
+# https://github.com/KSP-CKAN/CKAN/wiki/Releasing-a-new-CKAN-client#when-merging-pull-requests
+
+import sys
+from pathlib import Path
+from subprocess import run
+from exitstatus import ExitStatus
+import click
+from click import command, option, argument
+from git import Repo
+from github import Github
+
+@command()
+@option('--repo-path', type=click.Path(exists=True, file_okay=False),
+        default='.', help='Path to CKAN working copy')
+@option('--token', required=False, envvar='GITHUB_TOKEN')
+@argument('pr_num', type=click.INT)
+def merge_pr(repo_path: str, token: str, pr_num: int) -> None:
+    r = Repo(repo_path)
+    pr = Github(token).get_repo('KSP-CKAN/CKAN').get_pull(pr_num)
+    # Make sure master is checked out
+    if r.head.ref.name != 'master':
```

Comment:
Apparently `r.head.ref` throws an exception if you're in a detached HEAD state:
```
TypeError: HEAD is a detached symbolic reference as it points to 'b1d99eb25ce965ea6456f6b4fb59f07bb46c3aba'
```
The following seems to work fine, so the user gets a nice err message, not a stack trace:
```suggestion
    if r.head.is_detached or r.head.ref.name != 'master':
```